### PR TITLE
feat: Add labels for test result graph

### LINKED ISSUE #174: Labels on test result graph
Make it easier for the user to understand which data belongs to which ear on the test result graph.

### PR DESCRIPTION
**Pull request by dansvend**
This makes it easier for the user to understand which ear the data in the test result graph belongs to.

Closes #174

### CODE REVIEW

File: components/common/atoms/EarLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Typography } from "mad-expo-core";
+import React from "react";
+import { StyleProp, StyleSheet, View, ViewStyle } from "react-native";
+
+import { LEFT_EAR, RIGHT_EAR } from "../../../constants/colors";
```
Comment (ofredstie) [inline components/common/atoms/EarLabel.tsx:5]: Kunne du ikke sendt inn fargen som en prop og dermed sluppet import av colors i denne komponenten?

File: components/common/molecules/TestResultItem.tsx [hunk 2]
```diff
@@ -2,11 +2,12 @@ import { Typography } from "mad-expo-core";
 import React, { useEffect, useState } from "react";
 import { ScrollView, StyleSheet, View } from "react-native";
 
+import { Chart } from "./Chart";
 import { MOSS_GREEN_100, TEXT } from "../../../constants/colors";
```
Comment (ofredstie) [inline components/common/molecules/TestResultItem.tsx:6]: Og heller lagt til LEFT_EAR / RIGHT_EAR her

**Review by ofredstie — APPROVED** [review]
👍

File: components/common/atoms/EarLabel.tsx (hunk 1 above)
Comment (ofredstie) [reply components/common/atoms/EarLabel.tsx:5]: Hadde du hatt farge og tekst som prop så ville det vel ikke være nødvendig med de if-sjekkene

Comment (dansvend) [reply components/common/atoms/EarLabel.tsx:5]: I could send `color` as a prop, but that would mean you would need to import colors every where you use the component and it's easier for colors to be inconsistent that way. If I would make this component a generic component that would be part of a common component library I'd do what you suggested.

File: components/common/molecules/TestResultItem.tsx (hunk 2 above)
Comment (dansvend) [reply components/common/molecules/TestResultItem.tsx:6]: See other comment ☝️

**Pull request merged**